packages/razzle-plugin-scss/README.md

```diff
@@ -16,4 +16,185 @@ Using the plugin with the default options
 module.exports = {
   plugins: ['scss'],
 };
-```
\ No newline at end of file
+```
+
+### With custom options
+
+Please remember that custom options will extends default options using `Object.assign`.
+Array such as postcss.plugins __WILL NOT BE EXTENDED OR CONCATED__, it will override all default plugins.
+
+```js
+// default options
+const defaultOptions = {
+  postcss: {
+    dev: {
+      sourceMap: true,
+      ident: 'postcss',
+    },
+    prod: {
+      sourceMap: false,
+      ident: 'postcss',
+    },
+    plugins: [
+      PostCssFlexBugFixes,
```

Comment:
Not sure I'm in love with this API. Let me think about this for a few hours and come back with some ideas.